ico/reader.go

```diff
@@ -147,6 +147,8 @@ func (d *decoder) decodeHeader(r io.Reader) error {
 	return nil
 }
 
+var NoEntries = fmt.Errorf("no entries")
```

Comment:
No need to use fmt for such a simple error. I also think the error name doesn't follow the standard. Exported variables and functions should also always have a comment. 
```go
var ErrorNoEntires = errors.New("no entries")
```

Reply:
I opted for fmt because it is already imported and used by the `decodeHeader` function above.
Should i go for errors.New anyway?

Reply:
You should only use fmt.Errorf if you need to format the string. Static strings should use errors.New instead. It is such a minimal package that importing it is not a problem :)

Reply:
Done

Reply:
Please don't mark other people's comments as resolved. It is up to the reviewer to decide when they have been resolved.
The following hasn't been resolved:

> Exported variables and functions should also always have a comment.

Reply:
Something like `Returned when the gif contains no images`?

Reply:
It needs to follow the standard for Go comments `NameOfExportedFunction does something bla bla bla.` and not mention `gif` in this case of course. Maybe something along these lines:
```
// ErrorNoEntries is returned when the decoded image contains no entries.
```